onboarding/javascript.md

```diff
@@ -1,6 +1,6 @@
 ## Installation
 
-To use the JavaScript <a href="https://github.com/rollbar/rollbar.js" target="_blank" rel="noopener">rollbar.js</a> notifier, simply copy-paste the following code 
+To use the Javascript <a href="https://github.com/rollbar/rollbar.js" target="_blank" rel="noopener">rollbar.js</a> notifier, simply copy-paste the following code
```

Comment:
I think it really is `JavaScript` (capital S).

javascript.com: https://www.javascript.com/
mozilla: https://developer.mozilla.org/en-US/docs/Web/JavaScript
wikipedia: https://en.wikipedia.org/wiki/JavaScript
douglas crockford: http://javascript.crockford.com/javascript.html
microsoft: https://docs.microsoft.com/en-us/scripting/javascript/reference/javascript-reference